polygeist/tools/cgeist/Test/Verification/sycl/functions.cpp

```diff
@@ -218,10 +219,6 @@ SYCL_EXTERNAL void id_get_2(const sycl::id<1> id, int dimension) {
 // CHECK-MLIR:           %arg0: memref<?x!sycl_id_1_> {llvm.align = 8 : i64, llvm.byval = !sycl_id_1_, llvm.noundef}
 // CHECK-MLIR: %{{.*}} = "sycl.id.get"(%{{.*}}) {BaseType = memref<?x!sycl_id_1_, 4>, FunctionName = @"operator unsigned long", MangledFunctionName = @_ZNK4sycl3_V12idILi1EEcvmEv, TypeName = @id} : (memref<?x!sycl_id_1_, 4>) -> i64
 
-// CHECK-LLVM-LABEL: define spir_func void @_Z8id_get_3N4sycl3_V12idILi1EEE(
```

Comment:
Some CHECK-LLVM lines are removed, as calls are inlined, so not very useful to check.